test/unit/index.js

```diff
@@ -1,5 +1,6 @@
 import Vue from 'vue'
-
+// import Vuetify from 'vuetify';
+// Vue.use(Vuetify);
```

Comment:
Please remove unused code.

Reply:
This should have fixed many of the Vuetify (innocent) warnings for the tests, obscuring results, but it didn't somehow....